tests/rados/rados_prep.py

```diff
@@ -322,6 +343,113 @@ def create_erasure_pool(node: CephAdmin, name: str, **kwargs) -> bool:
     return True
 
 
+def enable_pg_autoscaler(node: CephAdmin, **kwargs) -> bool:
+    """
+    Configures pg_Autoscaler as a global global parameter and on pools
```

Comment:
The method name and the implementation is mismatching... could be rename the method name to convey the right message.

Reply:
changed